### PR TITLE
Create Toggle Mechanism and Remove Duplicate Button Code

### PR DESCRIPTION
**Pull request by knittingarch**
I created this branch off of the working branch for the toggles.  As a result, all of the toggle commits came with.  Didn't realize that would happen when I made this branch.  If you'd prefer that this PR only be about the duplicate code, I am happy to figure out how to do that.

The extracted button code (final commit) was in two files:
- key_value_mapping.html
  - included in corehq/apps/app_manager/templates/app_manager/module_view.html
- nav_menu_media_single_type.html
  - included in corehq/apps/app_manager/templates/app_manager/partials/nav_menu_media.html

I did a spot test on the Survey - Form Settings page, and all seemed to work as before, but more thorough functional testing by someone who knows the codebase a bit better would be wise :)

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -1344,3 +1344,10 @@ def _commtrackify(domain_name, toggle_is_enabled):
     TAG_INTERNAL,
     [NAMESPACE_USER]
 )
+
+NEW_MULTIMEDIA_UPLOADER = StaticToggle(
+    'new_multimedia_uploader',
+    'Display new multimedia uploader',
+    TAG_INTERNAL,
+    [NAMESPACE_DOMAIN]
+)
```
[the remark this answers is not in the document]
Comment (knittingarch) [reply corehq/toggles.py]: This change was made on the original branch.  Not sure why all the other changes from that branch carried over, but I'm assuming that it was a result of creating this branch based on that one.  What is the best way to get changes made after the branch creation onto this branch? Hope this makes sense...  Thanks!

**Comment by orangejenny** [comment]
> I created this branch off of the working branch for the toggles.

Yeah, please do update this to branch off master instead. Check out https://stackoverflow.com/questions/34068903/i-accidently-branched-off-of-a-branch-now-i-want-to-undo-this / https://git-scm.com/docs/git-rebase

**Comment by knittingarch** [comment]
@orangejenny I tried rebasing.  Now I can't push my changes.  Keep getting a fast-forward error even though I already did a `git pull`.  Feels a bit circular... Not quite sure how to resolve this issue.

**Comment by orangejenny** [comment]
Yeah, rebasing means you need to force push (`git push -f`), because the original commits are overwritten.

**Comment by knittingarch** [comment]
@orangejenny I'm still getting the same error even after force pushing.  I did a fresh pull to see if that would help, but no go.  Any other suggestions?

**Comment by snopoke** [comment]
Closing old PRs. Reopen if necessary.

**Pull request closed without merging**